.github/workflows/test.yml

```diff
@@ -1,6 +1,10 @@
 name: Testing CI
 
-on: pull_request
+on:
```

Comment:
I think this should be done at least in the separate commit since it doesn't fall under `fix: badge with CI status in readme`.

Reply:
Though they are useless without one another, the CI on master is useless on its own, the badge will not work without the CI changes.